helpers/hstring.py

```diff
@@ -94,6 +94,48 @@ def get_docstring_line_indices(lines: List[str]) -> List[int]:
     return docstring_line_indices
 
 
+# TODO(allenmatt10): Add tests.
+def get_function_indices(lines: List[str]) -> List[int]:
+    """
+    Get indices of lines of code that are inside functions and methods.
```

Comment:
let's add a bit of an example since we will forget what we are doing in 6 months